arangod/Metrics/ClusterMetricsFeature.cpp

```diff
@@ -262,11 +277,15 @@ bool ClusterMetricsFeature::readData(futures::Try<LeaderResponse>&& raw) {
     return false;
   }
   velocypack::Slice metrics{raw.get()->data()};
+  if (metrics.isNull()) {
+    return true;  // our data is up to date
+  }
```

Comment:
It's main part of fix.
If we receive null (none in old code, null used now because it more unique than zero byte)
it means our data is up to date and we don't need to try to repeat fetch data from leader